test/test_transforms_v2.py

```diff
@@ -186,7 +187,20 @@ def _check_functional_scripted_smoke(functional, input, *args, **kwargs):
         functional_scripted(input.as_subclass(torch.Tensor), *args, **kwargs)
 
 
-def check_functional(functional, input, *args, check_scripted_smoke=True, **kwargs):
+def _check_functional_torch_compile_smoke(functional, input, *args, **kwargs):
+    """Checks if the functional can be torch compiled and the compiled version can be called without error."""
+    if not isinstance(input, torch.Tensor):
+        return
```

Comment:
I thought of doing `type(input) == Tensor` instead of `isinstance`, so that we don't duplicate this check for subclasses.

Reply:
I think we need to be able to compile functional ops for subclasses as well. Right now dynamo does not fully support subclasses and in our case it can compile and even exec correctly if we just skip the last wrap to subclass.